definitions/infra-awsrekognition/dashboard.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "AWS Rekognition",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS Rekognition",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Request Counts",
```

Comment:
For consistency, the title should be "Request count"

Reply:
addressed